src/mpcCoreKit.ts

```diff
@@ -878,13 +882,13 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     const existingUser = await this.isMetadataPresent(this.state.postBoxKey);
     let importTssKey = providedImportTssKey;
     if (!existingUser) {
-      if (!importTssKey && !this.options.useDkg) {
+      if (!importTssKey && !this.useDKG) {
         if (this.keyType === KeyType.ed25519) {
           const k = generateEd25519Seed();
           importTssKey = k.toString("hex");
         } else if (this.keyType === KeyType.secp256k1) {
           const k = secp256k1.genKeyPair().getPrivate();
-          importTssKey = scalarBNToBufferSEC1(k).toString("hex");
+          importTssKey = scalarBNToBufferSEC1(k).toString("hex", 64);
```

Comment:
64 is not needed here. `scalarBNToBufferSEC1` already ensures it has correct length

Reply:
thanks i will keep in mind